tt_metal/soc_descriptors/wormhole_b0_versim.yaml

```diff
@@ -10,7 +10,7 @@ arc:
   [ ]
 
 pcie:
-  [ ]
+  [ 0-0 ]
```

Comment:
does the WH simulated soc have pcie cores? If not then I think we should remove this

Reply:
Yep should've removed this too, ty. 

Removed now.